liquibase-core/src/main/java/liquibase/parser/core/java/ChangeSet.java

```diff
@@ -36,7 +36,7 @@
      *
      * @return ordering
      */
-    String order();   // must be set
```

Comment:
I changed some of the fields from strings to more specific types. Maybe lose some ability to do parameter expansion, though? But also a bit easier to write. Unsure on which is better but changed it to think about.

Reply:
I agree, it will avoid some possible mistake